mapwms.c

```diff
@@ -959,9 +956,9 @@ int msWMSLoadGetMapParams(mapObj *map, int nVersion,
   outputFormatObj *format = NULL;
   int validlayers = 0;
   char *styles = NULL;
-  int numlayers = 0;
+  int numwmslayerarg = 0;
```

Comment:
Ideally renaming this variable would have been in a separate pull request and it would have been called "numwmslayerargs" (plural)... but we'll accept it as is for this time.

Reply:
Changs will be made